inspect/src/lib.rs

```diff
@@ -127,6 +138,60 @@ pub fn run(options: Options) -> anyhow::Result<()> {
             "{}",
             serde_json::to_string_pretty(&payload)?.to_colored_json_auto()?
         );
+    } else if options.history {
+        let branch = Reference::try_from(&urn)?;
+        match storage.reference(&branch) {
+            Ok(Some(reference)) => {
+                let mut tip = reference.peel_to_commit()?;
+
+                loop {
+                    let tree = tip.tree()?;
+                    let entry = tree.get(0).unwrap().id();
+                    let blob = storage
+                        .find_object(Box::new(entry))?
+                        .unwrap()
+                        .into_blob()
+                        .unwrap();
+                    let content: serde_json::Value = serde_json::from_slice(blob.content())?;
+                    let timezone = if tip.time().sign() == '+' {
+                        FixedOffset::east(tip.time().offset_minutes() * 60)
+                    } else {
+                        FixedOffset::west(tip.time().offset_minutes() * 60)
+                    };
+                    let time = DateTime::<Utc>::from(
+                        std::time::UNIX_EPOCH
+                            + std::time::Duration::from_secs(tip.time().seconds() as u64),
+                    )
+                    .with_timezone(&timezone)
+                    .to_rfc2822();
+
+                    println!(
+                        "{}  ▲\n  ┃\n  ┃\n",
+                        term::Box(format!(
+                            "tree {}\ncommit {}\nblob {}\ndate {}\n\n{}",
+                            term::format::highlight(term::format::bold(tree.id())),
+                            term::format::highlight(term::format::bold(tip.id())),
+                            term::format::highlight(term::format::bold(blob.id())),
+                            term::format::highlight(term::format::bold(time)),
+                            serde_json::to_string_pretty(&content)?.to_colored_json_auto()?,
+                        ))
+                    );
+
+                    if tip.parent_count() == 0 {
+                        break;
+                    }
+
+                    tip = tip.parent(0).unwrap();
```

Comment:
Why not combine this parent lookup with the `break` above

Reply:
wdym?

Reply:
https://github.com/radicle-dev/radicle-cli/pull/78/commits/205ece9cbc8d2ea54cf213f892f2a8f1e71ce4ff#diff-68a7e8c16c74cba4f71df82b4d89eac330072db4fcb43a4bd7de8bf3163283d9R185-R188